.gitignore

```diff
@@ -5,6 +5,7 @@ neatlynx/__pycache__
 *.pyc
 .env/
 .env2.7/
+.venv*/
```

Comment:
This is specific to your dev environment, not sure this change should go in this PR